challenge-07/challenge-07.js

```diff
@@ -54,7 +63,12 @@ Só vamos mudar o range:
 - No segundo "for", mostre os números ímpares entre 111 e 125, inclusive eles.
 */
 console.log( 'Números pares entre 100 e 120:' );
-// ?
+for( var num = 100; num <= 120; num++){
+	num % 2 === 0 ? console.log(num++) : '';
```

Comment:
Acho que compreendi! Na verdade o <code>for</code> ele testa por tudo! Só que os ímpares ele não mostra, certo ?! :)